orquesta/tests/unit/fixture_mock/test_fixture_mock.py

```diff
@@ -0,0 +1,199 @@
+          - exception:
+              route: 0
+              status: "succeeded"
+        """
+        mock_open = mock.mock_open(read_data=wf)
+        if sys.version_info[0] < 3:
+            with mock.patch("__builtin__.open", mock_open):
+                spec = TestFileSpec(spec_yaml, "fixture")
+                workflow_path = "/tmp"
+                self.assertRaises(exc.WorkflowSpecError, Fixture, spec, workflow_path)
+        else:
+            with mock.patch("builtins.open", mock_open):
+                spec = TestFileSpec(spec_yaml, "fixture")
+                workflow_path = "/tmp"
+                self.assertRaises(exc.WorkflowSpecError, Fixture, spec, workflow_path)
+
+    @mock.patch("orquesta.tests.mocks.Fixture.load_wf_spec")
+    def test_run_test_throw(self, load_wf_spec):
+
+        wf_name = "sequential"
+        self.assert_spec_inspection(wf_name)
+        wf_def = self.get_wf_def(wf_name)
+        wf_spec = self.spec_module.instantiate(wf_def)
+
+        load_wf_spec.return_value = wf_spec
+        spec_yaml = """
+        workflow: "/tmp/test"
+        routes: [[]]
+        task_sequence:
+          - task1:
+              route: 0
+              status: "succeeded"
+              result: {}
+          - task2:
+              route: 0
+              status: "succeeded"
+              result: {}
+          - task3:
+              route: 0
+              status: "failed"
+              result: {}
+          - continue:
+              route: 0
+              status: "succeeded"
+              result: {}
+          - exception:
+              route: 0
+              status: "succeeded"
+              result: {}
+        """
+
+        spec = TestFileSpec(spec_yaml, "fixture")
+        workflow_path = "/tmp"
+        fixture = Fixture(spec, workflow_path)
+        self.assertRaises(exc.TaskEquality, fixture.run_test)
+
+    @mock.patch("orquesta.tests.mocks.Fixture.load_wf_spec")
+    def test_run_test(self, load_wf_spec):
+
+        wf_name = "sequential"
+        self.assert_spec_inspection(wf_name)
+        wf_def = self.get_wf_def(wf_name)
+        wf_spec = self.spec_module.instantiate(wf_def)
+        load_wf_spec.return_value = wf_spec
+        spec_yaml = """
+        workflow: "/tmp/test"
+        routes: [[]]
+        task_sequence:
+          - task1:
+              route: 0
+              status: "succeeded"
+              result: {"result":{"test":true}}
+          - task2:
+              route: 0
+              status: "succeeded"
+              result: {"result":{"test":true}}
+          - task3:
+              route: 0
+              status: "succeeded"
+              result: {"result":{"test":true}}
+          - continue:
+              route: 0
+              status: "succeeded"
+              result: {"result":{"test":true}}
+        """
+
+        spec = TestFileSpec(spec_yaml, "fixture")
+        workflow_path = "/tmp"
+        fixture = Fixture(spec, workflow_path)
+        fixture.run_test()
```

Comment:
I don't see any unit test where the test fixtures are actual files (fixture and results in separate files) are on the file system and the unit tests load them from files.